test/inductor/test_flex_decoding.py

```diff
@@ -27,6 +28,15 @@
     flex_attention_supported_platform as supported_platform,
     instantiate_device_type_tests,
 )
+from torch.testing._internal.common_utils import IS_CI, IS_WINDOWS
+
+
+if IS_WINDOWS and IS_CI:
+    # TODO(xuhancn) : Need track if it is a requirement on windows.
+    sys.stderr.write("This UT is validated on windows, a lot of crash. Skip it.\n")
```

Comment:
Do you happen to know why FlexDecoding crashes on windows?

Reply:
> Do you happen to know why FlexDecoding crashes on windows?

For example: 'test/inductor/test_flex_decoding.py::TestFlexDecodingCPU::test_non_pow_2_headdim_head_dim_121_float32_cpu_float32'
The reason for crash is `divide by zero error`, but msvc's PDB is too much optimize to line number level debug. Not more debug work due to it is not target feature.

Reply:
I see, thanks!